src/main/java/org/gridsuite/actions/server/ContingencyListController.java

```diff
@@ -111,6 +111,16 @@ public ResponseEntity<List<ContingencyInfos>> exportContingencyInfosList(@PathVa
         return ResponseEntity.ok().body(service.exportContingencyInfosList(id, networkUuid, variantId));
     }
 
+    @GetMapping(value = "/contingency-lists/contingency-infos/export", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Evaluate and export a contingency infos list to JSON format")
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The contingency list in JSON format"),
+        @ApiResponse(responseCode = "404", description = "The contingency list does not exists")})
+    public ResponseEntity<List<ContingencyInfos>> exportContingencyInfosList(@RequestParam(value = "networkUuid", required = false) UUID networkUuid,
```

Comment:
we never return (responseCode = "404") from this endpoint, to remove ?

Reply:
404 can happen in 
```
ContingencyListService.java:179
.orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND, "Contingency list " + id + " not found"));
```